packages/components/src/components/location-info/location-info.tsx

```diff
@@ -59,6 +59,11 @@ export class LocationInfo implements ComponentInterface {
             details.push(this.location.properties.subtitle);
         }
 
+        // Place formatter (For Mapbox usage)
+        if (this.location.properties.place_formatted) {
+            details.push(this.location.properties.place_formatted);
```

Comment:
There is no similar handling for Google? 🤔

Reply:
That's true, this check is not necessary anymore. Removed the added code ✅